spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/OpenTelemetryAutoConfigurationTests.java

```diff
@@ -192,16 +191,6 @@ void shouldSupplyW3CPropagationWithoutBaggageWhenDisabled() {
 				.run((context) -> assertThat(context).hasBean("w3cTextMapPropagatorWithoutBaggage"));
 	}
 
-	@Test
-	void shouldSupplyB3PropagationWithoutBaggageWhenBaggageDisabledAndB3PropagationEnabled() {
```

Comment:
I removed this test method as it seems to be a duplicate of `shouldSupplyB3PropagationIfPropagationPropertySetAndBaggageDisabled()`.